internal/resources/aws/ebs_snapshot.go

```diff
@@ -0,0 +1,140 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+
+	"github.com/shopspring/decimal"
+)
+
+type EbsSnapshot struct {
+	Address                  string
+	Region                   string
+	VolumeRefSize            float64
```

Comment:
```suggestion
	SizeGB            *float64
```
We should probably update the `ebs_volume` resource to be consistent. Also should this be a *float so we default to `defaultVolumeSize` only if it's nil?

Reply:
Sure, fixed.